src/callbacks.rs

```diff
@@ -31,6 +31,12 @@ pub trait ParseCallbacks: fmt::Debug + UnwindSafe {
         MacroParsingBehavior::Default
     }
 
+    /// This function will run for every function. The returned value determines the name visible
+    /// in the bindings.
+    fn link_name_override(&self, _function_name: &str) -> Option<String> {
```

Comment:
I'm confused, I'd expect this to change the `link_name`, but instead it seems it changes the generated name, right?

To be clear, that's also something that would solve the issue, but then the name is wrong isn't it?

Reply:
Right, the intention is to change the generated name (as it appears to Rust code). I used "link_name_override" based on guidance provided in the issue this PR is addressing.

Perhaps the name should be "generated_name_override"? I'm open to suggestions.